intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/problems/DelegatingPluginCreationResultResolver.kt

```diff
@@ -0,0 +1,24 @@
+package com.jetbrains.plugin.structure.intellij.problems
+
+import com.jetbrains.plugin.structure.base.plugin.PluginCreationResult
+import com.jetbrains.plugin.structure.base.plugin.PluginProblem
+import com.jetbrains.plugin.structure.intellij.plugin.IdePlugin
+
+abstract class DelegatingPluginCreationResultResolver(protected val delegatedResolver: PluginCreationResultResolver) : PluginCreationResultResolver {
```

Comment:
Shouldn't `LevelRemappingPluginCreationResultResolver` inherit `DelegatingPluginCreationResultResolver`? Otherwise, I don't understand why `DelegatingPluginCreationResultResolver` is needed.

Reply:
Removed `DelegatingPluginCreationResultResolver` altogether, as there is only a single inheritor in test suite.